src/bp/ui-studio/src/web/views/OneFlow/sidePanel/GoalEditor/index.tsx

```diff
@@ -61,6 +60,19 @@ const EditGoalModal: FC<Props> = props => {
     }
   }, [props.isOpen])
 
+  const getNameOnly = (name: string): string => {
+    const explodedName = name.split('/')
+
+    return explodedName[explodedName.length - 1].replace(/\.flow\.json$/i, '')
```

Comment:
afraid of that one liner  >< ? 

`return name.split('/').splice(-1).replace(/\.flow\.json$/i, '')`

Reply:
Will fix in my next branch